### PR TITLE
Accessibility amends for non-descriptive headings

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3155

### Change description ###
Amend non-descriptive headings for accessibility

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Daniel-Lam — APPROVED** [review]
Looks OK to me, @paulhowes-HMCTS

File: src/shared/components/search-result/search-result.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <table *ngIf="hasResults() || hasDrafts()">
   <caption>
-    <h2 class="heading-h2">{{ caseState ? 'Case List' : 'Search result' }}</h2>
+    <h2 class="heading-h2">{{ caseState ? 'Your cases' : 'Search result' }}</h2>
```
Comment (paul-graham) [inline src/shared/components/search-result/search-result.component.html:3]: Being picky, shouldn't the false state say 'Search results' to maintain the plural nature? For the sake of consistency.

**Review by paul-graham — APPROVED** [review]

**Review by adnanakgun — APPROVED** [review]

File: src/shared/components/search-result/search-result.component.html (hunk 1 above)
Comment (paulhowes-HMCTS) [reply src/shared/components/search-result/search-result.component.html:3]: Thank's Paul I wasn't asked to change that however I'll mention your comment on the ticket and include the necessary people to bring it to their attention.

**Comment by paulhowes-HMCTS** [comment]
Superceded by https://github.com/hmcts/ccd-case-ui-toolkit/pull/794 so closing.

**Pull request closed without merging**